src/styles/blocks/card.scss

```
@@ -0,0 +1,114 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
```

move `card-width` to the variable
```suggestion
  width: $...;
```